build.psake.ps1

```diff
@@ -120,6 +120,9 @@ Properties {
     $PublishRootDir = "$PSScriptRoot\.publish"
     $PublishDir     = "$PublishRootDir\$ModuleName"
 
+    # The local installation directory for the install task. Defaults to your user PSModulePath.
+    $InstallPath = Join-Path -Path (Split-Path $profile.CurrentUserAllHosts -Parent) -ChildPath $ModuleName 
+
```

Comment:
That ChildPath should be at least `"Modules\$ModuleName"` and if we have the version number, perhaps `"Modules\$ModuleName\$ModuleVersion"` but I'm not sure if we have that. We could peek into the PSD1 file to pull it out?  Actually, this brings up another issue I want to address - version number management.